### ISSUE TITLE
Test data for Development Unblocking: Put Fake data in QA to develop against.

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/roadmap#741]
Following #43: https://github.com/icgc-argo/argo-metadata-schemas/issues/43

- [ ] Populate a NEW fake data index for the file repository to unblock development. 
- Should include `release_stage` and `clinical` data. 
- [ ]  Add the data to `dev` and `qa` environments. 

# Steps to execute 
- update mapping to new mapping for `file_centric` index 
- write a set of curl commands that will input the data into the index. 
- add values in a subset of documents for these fields

**Comment by samrichca** [comment]
ticket update: going to make an index with Rosi's test data

**Comment by rosibaj** [comment]
no longer needed
